### ISSUE TITLE
Export csv

### ISSUE THREAD

**Issue by orgogliobauscia** [akasolace/HO#77]
Would it be possible to have Personality, Aggressivity and Honesty as numbers and not as words (hope you understand what I mean).
I have also realized that the columns' order in the cvs file is different from the one I downloaded last week (using HO 1.435 stable).
Thanks :-)

**Comment by akasolace** [comment]
> Would it be possible to have Personality, Aggressivity and Honesty as numbers and not as words (hope you understand what I mean).

as far as I remember it would be consistent with characteristics so it would make sense
Does anyone in the development team object to that request ?


> I have also realized that the columns' order in the cvs file is different from the one I downloaded last week (using HO 1.435 stable).

This has to be linked to closing of issue #42 

> Thanks :-)

it is a small one so I guess it can make it on time for 1.436

**Comment by mistermax80** [comment]
> Does anyone in the development team object to that request ?

I introduced words in respect to numbers, I thought it was an improvement.
I would also specialities change in words.
For me ok, rollback to numbers.
For the order, if I don't remeber bad, I aligned order column to squad tab.

**Comment by akasolace** [comment]
@mistermax80 do you export translated words or alwawys in English ?

**Comment by akasolace** [comment]
@mistermax80   maybe in export csv, we can offer to export either as number or as translated string.  Hence, it pleases everyone. What do you think? 

@orgogliobauscia  having half of the export as numbers and the other half as text is really a question of taste. I am sure everyone has a different opinion on this

**Comment by mistermax80** [comment]
> @mistermax80 do you export translated words or alwawys in English ?

the words exported (words and headers) are translated

> @mistermax80 maybe in export csv, we can offer to export either as number or as translated string. Hence, it pleases everyone. What do you think?

Yes, It's a good Idea... I will can do it

**Comment by orgogliobauscia** [comment]
Thanks for your interest in the matter.

**Comment by tychobrailleur** [comment]
Does it matter if both are included in two different columns?  Sounds like the easiest solution for this to me.

**Comment by akasolace** [comment]
I think it concerns only columns  Agreeability, Aggressiveness and Honesty. I think we can simply export for those 3 columns both text and numerical values like "balanced(4)" and leaving the remaining unchanged

**Comment by tychobrailleur** [comment]
> I think it concerns only columns Agreeability, Aggressiveness and Honesty. 

Exactly.  So I am thinking adding an extra 3 columns holding those values as number would not be a big deal, and would allow people to sort on this values.

**Comment by akasolace** [comment]
yes either one additional column or as I suggested above only one column numeric + text  (that will also allow sorting) ...
